content/docs/command-reference/exp/diff.md

```diff
@@ -0,0 +1,39 @@
+# exp diff
+
+Show [parameters](/doc/command-reference/params) and
+[metrics](/doc/command-reference/metrics) changes between `dvc experiments`, or
+between the last <abbr>project</abbr> version and the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc experiments diff [-h] [-q | -v] [--all] [--show-json]
+                            [--show-md] [--old] [--no-path]
+                            [--precision <n>] [a_rev] [b_rev]
+
+positional arguments:
+  a_rev            Old experiment to compare (defaults to HEAD)
+  b_rev            New experiment to compare (defaults to the current
+                   workspace)
+```
+
+# Description
+
+Provides a quick way to compare `dvc params` and `dvc metrics` between two
+`dvc experiments`. The differences shown by this command include the old and new
+values, and numeric difference (delta) from the previous ones.
+
+> This is similar to combining the reports from `dvc params diff` and
+> `dvc metrics diff` together, for the experiments in question.
+
+Without arguments, this command compares params and metrics currently present in
+the <abbr>workspace</abbr> (last `dvc exp run`) with the latest committed
+versions (required).
```

Comment:
Is this correct?

Reply:
Without arguments it just compares whatever is present in the workspace (which may or may not be an experiment run) against git HEAD, same as metrics/params diff

Reply:
OK so this seems correct ✔️

I'm not sure about that behavior though, What about comparing the last experiment with its base by default? Cc @shcheklein 👋

Reply:
What is the base? How would we define that? It seems like that's going to depend on a user's experiment design, and the current behavior is about as good a guess as we could make about what commit would be the base. It's consistent with all the other dvc commands as @pmrowla mentioned, so my vote would be to leave it as is.

Reply:
@dberenbaum the base of `dvc exp run` is always the last committed project version at the time of the run AFAU. They are actual Git commits (in the `exps` custom ref) so you can just look at the commit parent to determine the base, I think.

Reply:
WAIT... 

> What about comparing the last experiment with its base

I guess that's what the current behavior does if we assume the workspace has the last `exp run` 🙂 

The question could be whether to comp the last `exp` (by timestamp) to it's parent but that doesn't seem more inuitive.

OK nvmd! Resolving this one, thanks.